src/anbox/qemu/sensors_message_processor.cpp

```diff
@@ -17,24 +17,64 @@
 
 #include "anbox/qemu/sensors_message_processor.h"
 
+#include <fmt/core.h>
+#include <fmt/format.h>
+
+#include <boost/algorithm/string.hpp>
 #include <boost/algorithm/string/predicate.hpp>
 #include <chrono>
 #include <iostream>
 #include <thread>
 
+#include "anbox/application/sensor_type.h"
 #include "anbox/logger.h"
 
 using namespace std;
+using namespace anbox::application;
+
+template <>
+struct fmt::formatter<std::tuple<double, double, double>> {
+  template <typename ParseContext>
+  constexpr auto parse(ParseContext& ctx) {
+    return ctx.begin();
+  }
+
+  template <typename FormatContext>
+  auto format(std::tuple<double, double, double> const& tuple, FormatContext& ctx) {
+    return fmt::format_to(ctx.out(), "{0:f}:{1:f}:{2:f}", std::get<0>(tuple), std::get<1>(tuple), std::get<2>(tuple));
+  }
+};
 
 namespace anbox {
 namespace qemu {
 SensorsMessageProcessor::SensorsMessageProcessor(
     shared_ptr<network::SocketMessenger> messenger, shared_ptr<application::SensorsState> sensorsState)
     : QemudMessageProcessor(messenger), sensors_state_(sensorsState) {
+  enabledSensors_ = 0;
   thread_ = std::thread([this]() {
     for (;;) {
-      if (temperature_.load())
-        send_message("temperature:" + to_string(sensors_state_->temperature));
+      auto enabledSensors = enabledSensors_.load();
+      if (enabledSensors & SensorType::AccelerationSensor)
+        send_message(fmt::format("acceleration:{0}", sensors_state_->acceleration));
```

Comment:
Any reason we're using fmt::format when we already have `string_format` in `src/anbox/utils.h`? Using different implementations for the same thing isn't what we should do.

Reply:
Hmm... Reason was that I didn't know about `string_format`. I can refactor if required. On the other hand seems like libfmt is the future of string formatting in C++ that made it into recent standard (C++20). Decision is yours.

Reply:
@morphis, how do we proceed with this PR? Should I make the changes to use `string_format` or can it be merged as is?

Reply:
I would like our code to be consistent. So either we convert evertyhing to `std::format` or we stick to `string_format` for the time being. Given the amount of work a conversion is, I would like to stick to `string_format` in this PR and then we can move globally to something else.

Reply:
Done.